static/js/controllers/reports.js

```diff
@@ -250,17 +335,18 @@ var _ = require('underscore'),
             _query();
           }
         }
-
       };
 
       $scope.$on('ClearSelected', function() {
-        $scope.selected = null;
+        $scope.selected = [];
+        $('#reports-list input[type="checkbox"]')
+          .prop('checked', false);
         liveList.clearSelected();
```

Comment:
Hey, should the livelist selection always match the checkbox selection? If yes, how could we wrap them together so that they always happen together?